ballista/scheduler/src/scheduler_server/mod.rs

```diff
@@ -155,7 +156,7 @@ impl<T: 'static + AsLogicalPlan, U: 'static + AsExecutionPlan> SchedulerServer<T
     pub fn running_job_number(&self) -> usize {
         self.state.task_manager.running_job_number()
     }
-
+    #[allow(dead_code)] // used by rest-api feature
```

Comment:
can we not do `#[cfg(feature = "rest-api")]` here?

Reply:
we absolutluy could, will fix